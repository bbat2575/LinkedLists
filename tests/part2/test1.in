NEW 1
1
INSERT 0 1 2
INSERT 0 2 3
INSERT 0 1 +
INSERT 0 3 =
TYPE 0
NEW 0
INSERT 1 0 Once
INSERT 1 1 a
INSERT 1 2 time
INSERT 1 -3 upon
INSERT 1 4 ...
TYPE 1
NEW 0
INSERT 2 0 22.22
INSERT 2 -2 2e4
INSERT 2 2 0.1
INSERT 2 -1 2e-4
TYPE 2
NEW 0
INSERT 3 0 
INSERT 3 0  
INSERT 3 0   
INSERT 3 0    
TYPE 3
NEW 0
INSERT 4 0 99                                                                                                                  
INSERT 4 1                                                                                                                   99
INSERT 4 2 9.9                                                                                                                 
INSERT 4 3                                                                                                                  9.9
INSERT 4 -1 $                                                                                                                  
INSERT 4 -1                                                                                                                   $
INSERT 4 6 Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime
TYPE 4
VIEW ALL
