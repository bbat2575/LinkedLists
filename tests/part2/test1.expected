
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> List 0: 1 -> 2

> List 0: 1 -> 2 -> 3

> List 0: 1 -> + -> 2 -> 3

> List 0: 1 -> + -> 2 -> = -> 3

> int -> char -> int -> char -> int

> List 1: 

> List 1: Once

> List 1: Once -> a

> List 1: Once -> a -> time

> List 1: Once -> upon -> a -> time

> List 1: Once -> upon -> a -> time -> ...

> string -> string -> char -> string -> string

> List 2: 

> List 2: 22.22

> List 2: 20000.00 -> 22.22

> List 2: 20000.00 -> 22.22 -> 0.10

> List 2: 20000.00 -> 22.22 -> 0.10 -> 0.00

> float -> float -> float -> float

> List 3: 

> List 3: 

> List 3:   -> 

> List 3:    ->   -> 

> List 3:     ->    ->   -> 

> string -> string -> char -> string

> List 4: 

> List 4: 99

> List 4: 99 -> 99

> List 4: 99 -> 99 -> 9.90

> List 4: 99 -> 99 -> 9.90 -> 9.90

> List 4: 99 -> 99 -> 9.90 -> 9.90 -> $                                                                                                                  

> List 4: 99 -> 99 -> 9.90 -> 9.90 -> $                                                                                                                   ->                                                                                                                   $

> List 4: 99 -> 99 -> 9.90 -> 9.90 -> $                                                                                                                   ->                                                                                                                   $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> int -> int -> float -> float -> string -> string -> string

> Number of lists: 5
List 0
List 1
List 2
List 3
List 4

> 
